source/PlayniteSDK/Models/Game.cs

```diff
@@ -260,6 +262,24 @@ public List<Guid> GenreIds
             }
         }
 
+        private bool hdr;
+        /// <summary>
+        /// Gets or sets value indicating if the game is should enable HDR on the primary monitor while playing.
+        /// </summary>
+        public bool Hdr

```

Comment:
Should be called something like `EnableSystemHDR` to make it clear what it actually does at first glance.

Reply:
Done